store/pg/store.go

```diff
@@ -83,9 +83,9 @@ func (s *Store) Stats() (model.Stats, error) {
 	const query = `
 	SELECT
 		(SELECT COUNT(*) FROM acl_whitelist) as RegisteredUsers,
-		(SELECT COUNT(*)/2 FROM program) as ProofsGenerated,
-		(SELECT COUNT(*) FROM transaction WHERE kind = 'proof' AND executed IS TRUE) as ProofsGenerated,
-		(SELECT COUNT(*) FROM transaction WHERE kind = 'proof' AND executed IS TRUE) as ProofsVerified;`
+		(SELECT COUNT(DISTINCT(prover)) FROM deploy) as ProversDeployed,
+		(SELECT COUNT(*) FROM transaction WHERE kind = 'proof') as ProofsGenerated,
+		(SELECT COUNT(DISTINCT(t.hash)) FROM transaction AS t JOIN proof AS p ON p.parent = t.hash JOIN verification AS v ON p.tx = v.parent WHERE t.kind = 'run') as ProofsVerified;`
```

Comment:
This is a bit pessimistic number now, as it reports only individual Run jobs that have at least one proof that is verified (with one verification), but I think this is what is ultimately wanted. Especially when in the future with the consensus & everything, each proof job will be "distinct" (i.e. the race between provers that was originally planned is now changed into better approach).